api/envoy/extensions/filters/network/dubbo_proxy/v3/dubbo_proxy.proto

```diff
@@ -47,7 +61,19 @@ message DubboProxy {
   SerializationType serialization_type = 3 [(validate.rules).enum = {defined_only: true}];
 
   // The route table for the connection manager is static and is specified in this property.
-  repeated RouteConfiguration route_config = 4;
+  //
+  // .. note::
+  //
+  //   This field is deprecated. Please use ``drds`` or ``multiple_route_config`` first.
+  repeated RouteConfiguration route_config = 4
+      [deprecated = true, (envoy.annotations.deprecated_at_minor_version) = "3.0"];
```

Comment:
Sorry this should also be documented in the deprecated section of the release notes.

Reply:
Get it, thanks.